Create style-guide.md (#184)
w3c/wai-website-theme: _style-guide/grammar.md

title: "Grammar – WAI Style Guide"
title_sentence: "Grammar"
title_html: "Grammar"
nav_title: "Grammar"
lang: en
order: 4
---

{% include toc.html %}

## Capitalization

### Abbreviations

See [Capitalization in abbreviations](/style-guide/style-different-content-types/#capitalization-in-abbreviations).

### All caps

Avoid using all capital letters for words (except acronyms).

### Braille

Use lowercase for “braille” (unless you are referring to Louis Braille).

### Glossaries

Use lowercase for glossary terms.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

The following items in this glossary apply to all technologies:

<ul>
  <li>abbreviation</li>
  <li>alternative to time-based media</li>
  <li>audio</li>
</ul>
</div>

### Headings

See [Capitalization in headings](/style-guide/style-different-content-types/#capitalization-in-headings).

### Lists

See [Capitalization and punctuation in lists](/style-guide/style-different-content-types/#capitalization-and-punctuation-in-lists).

### Proper nouns

Only capitalize nouns that are proper nouns.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Examples:

> * Proper noun capitalized: “Success Criterion 1.1.1 is about non-text content.”

Reviewer: ```suggestion
* Proper noun capitalized: “The goal of Success Criterion 2.5.8 Target Size (Minimum) is to make controls easier to activate.”
```

Suggesting an example that does not contradict "Do not refer to success criteria by number alone. Include the title." in [WCAG success criteria](https://deploy-preview-184--wai-website-theme.netlify.app/style-guide/tone-language-words/#wcag-success-criteria).

I appreciate how this could work as an exception, though.

* Common noun lowercase: “One success criterion has been removed.”

</div>

#### Proper nouns that have become common nouns

Use lowercase for the following nouns, unless these are part of a name or title:

* web (the web)
* internet (the internet)

## That versus which

<b>That:</b> Introduces essential information needed to understand the sentence

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

“The meeting that starts at 10 a.m. will be online” — here, the time is important, as it helps to identify which meeting is online.
</div>

<b>Which:</b> Introduces extra information that is not essential.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

“The meeting, which starts at 10 a.m., will be online.” — here, the time is additional, non-essential information.
</div>

## Punctuation

### Colons

#### Colon

Use a colon to introduce the main idea(s). It gives the sense of “as follows.”

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Examples:

<ul>
  <li>Contact us: email@example.com</li>
  <li>They had one goal: to finish the marathon.</li>
</ul>
</div>

> Compare with the use of an [em dash](#em-dash), which clarifies or elaborates on the main idea that was just said.

Reviewer 2: ```suggestion
Compare with the use of an [em dash](#em-dash-), which clarifies or elaborates on the main idea that was just said.
```

</ul>
</div>

Use a hanging hyphen when two compound adjectives modify the same noun.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

The pre<b>-</b> and post<b>-</b>deployment stages are equally important.
</div>

For guidance on terms we no longer hyphenate and are written as one word, see the section on [Spelling](/style-guide/tone-language-words/#spelling).

### Ellipsis (...)

Use an ellipsis to show:

* missing words
* a pause
* something left unsaid

Add a space before and after an ellipsis.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

I was wondering … but it’s probably not important.
</div>

### Headings

See [punctuation in headings](/style-guide/style-different-content-types/#punctuation-in-headings).

### Links

See [punctuation in links](/style-guide/style-different-content-types/#punctuation-in-links).

### Lists

See [punctuation in lists](/style-guide/style-different-content-types/#capitalization-and-punctuation-in-lists).

### Numbers

See [punctuation in numbers](/style-guide/style-different-content-types/#punctuation-in-numbers).

### Parentheses

> Use parentheses sparingly for clarifiction.

Reviewer: ```suggestion
Use parentheses sparingly for clarification.
```

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

> - Captions (also called “intralingual subtitles”) provide content to people who are deaf and others who cannot hear the audio.

Reviewer: ```suggestion
Captions (also called “intralingual subtitles”) provide content to people who are deaf and others who cannot hear the audio.
```

Reviewer 3: Very strong -1 to this example. We do not use the term 'intralingual subtitles'. At best, this could be 'sometimes called subtitles'.

Reviewer: ping @tamsinewing555 @shawna-slh

In my suggestion, I only removed the leading `-` to avoid a one-item list.

</div>

Consider a separate sentence instead of complicating one sentence with parentheses.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

> - Captions provide content to people who are deaf and others who cannot hear the audio. Captions are also called “intralingual subtitles”.

Reviewer: ```suggestion
Captions provide content to people who are deaf and others who cannot hear the audio. Captions are also called “intralingual subtitles”.
```

Reviewer 3: Same as [above](https://github.com/w3c/wai-website-theme/pull/184#discussion_r3093611900)

Reviewer: ping @tamsinewing555 @shawna-slh

In my suggestion, I only removed the leading `-` to avoid a one-item list.

</ul>
</div>

Use a hanging hyphen when two compound adjectives modify the same noun.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

The pre<b>-</b> and post<b>-</b>deployment stages are equally important.
</div>

For guidance on terms we no longer hyphenate and are written as one word, see the section on [Spelling](/style-guide/tone-language-words/#spelling).

### Ellipsis

Use an ellipsis (...) to show:

* missing words
* a pause
* something left unsaid

Add a space before and after an ellipsis.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

I was wondering … but it’s probably not important.
</div>

### Headings

See [punctuation in headings](/style-guide/style-different-content-types/#punctuation-in-headings).

### Links

See [punctuation in links](/style-guide/style-different-content-types/#punctuation-in-links).

### Lists

See [punctuation in lists](/style-guide/style-different-content-types/#capitalization-and-punctuation-in-lists).

### Numbers

See [punctuation in numbers](/style-guide/style-different-content-types/#punctuation-in-numbers).

### Parentheses

> <p><span style="border: 5px solid var(--faded-red);"><strong style="background-color: var(--faded-red);border-right: 5px solid var(--faded-red); color: var(--pure-white);">Open Issue</strong> <a href="https://github.com/w3c/style-guide-v2025/issues/44">Issue #44</a> — Needs more simplification</span></p>

Reviewer 2: ```suggestion
```

> Use parentheses to clarify something in an aside:

Reviewer 2: ```suggestion
Use parentheses sparingly to clarify something in an aside:
```

</div>

Scenario 3: To signal an optional plural:

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

Test(s)
</div>

**Note:** Parenthetical asides can be distracting, so use them sparingly and, where it makes sense to, rewrite the content as separate sentences.

### Colons

#### Colon

Use a colon to introduce the main idea(s), giving the sense of "as follows".

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Examples:

<ul>
  <li>Contact us: email@example.com</li>
  <li>They had one goal: to finish the marathon.</li>
</ul>
</div>

Compare with the use of an [em dash](#em-dash), which clarifies or elaborates on the main idea that was just said.

#### Semicolon

Use a semicolon in the following scenarios.

Scenario 1: To connect two or more independent clauses not joined by a conjunction (that is, each clause can stand alone as a *grammatically complete* sentence):

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

Bananas and apples are both fruits; however, they do not look alike.
</div>

> Scenario 2: To break up complex lists that contain internal commas:

Reviewer: ```suggestion
Scenario 2: To break up complex in-line lists that contain internal commas:
```

Rationale: make it clear this does not apply to bullet lists.

### Colons

#### Colon

Use a colon to introduce the main idea(s). It gives the sense of “as follows.”

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Examples:

<ul>
  <li>Contact us: email@example.com</li>
  <li>They had one goal: to finish the marathon.</li>
</ul>
</div>

Compare with the use of an [em dash](#em-dash), which clarifies or elaborates on the main idea that was just said.

#### Semicolon

Use a semicolon in the following scenarios.

Scenario 1: To connect two or more independent clauses not joined by a conjunction (that is, each clause can stand alone as a *grammatically complete* sentence):

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

Bananas and apples are both fruits; however, they do not look alike.
</div>

Scenario 2: To break up complex lists that contain internal commas:

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

<ul>
  <li>Meeting attendees are from London, England; Tokyo, Japan; and São Paulo, Brazil.</li>
</ul>
</div>

### Commas

> In a phrase listing three or more items, place a comma before the final conjunction.

Reviewer 2: ```suggestion
In a phrase listing three or more items, place a comma before the final conjunction ("Oxford comma").
```

very mild suggestion. feel free to not do it.

rationale: Some people know "Oxford comma" so including it would help them understand this quicker.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

I bought apples, oranges, and bananas.
</div>

### Dashes and hyphens

#### Em dash

* Use an em dash (—) to clarify or elaborate on what was just said.
* Add a space before and after an em dash.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Examples:

<ul>
  <li>Many animals hibernate during winter — for example, bears.</li>
  <li>The number of new cases has been declining — last week’s daily average was the lowest since January.</li>
  <li>Make the web work — for everyone.</li>
</ul>
</div>

**Note:** Do not use an em dash to join related ideas if it makes the sentence long or complicated.

Compare with the use of a [colon](#colon), which introduces the main idea(s), giving the sense of "as follows".

#### En dash

> * Use an en dash (–) to indicate a range in numbers, such as in dates, pages and sports results.

Reviewer 2: ```suggestion
* Use an en dash (–) to indicate a range in numbers, such as in dates, pages, and sports results.
```

---
title: "Grammar"
lang: en
---

{% include toc.html %}

## Capitalization

### Abbreviations 

See [Capitalization in abbreviations](/style-guide/style-different-content-types/#capitalization-in-abbreviations).

### All caps 

Avoid using all caps for words (except acronyms).

### Braille

Use lowercase for braille (unless referring to the person Louis Braille).

### Glossaries

> Use lowercase for terms in a glossary.

Reviewer 3: ```suggestion
Use lowercase for glossary terms
```

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

<ul>
  <li>It's easy; see the <a href="/style-guide/structure-presentation/#structure">Structure section</a>.</li>
</ul>
</div>

#### Em dash (optional)

If you want to convey emphasis or a sharp break in thought, you can use an em dash with simple clauses. (Note that em dashes are associated with AI writing.)

The first part should be a complete sentence. The part after the dash does not need to be a complete sentence.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

<ul>
  <li>The W3C mission is to make the web work — for everyone.</li>
</ul>
</div>

See also [Em dashes](/style-guide/grammar/#em-dashes).

#### Parentheses (optional)

When related information applies to a point within a simple sentence, you can put the information in parentheses within the sentence. Be aware that parentheses break up the sentence flow and make the sentence more complex.

<div class="inset-text inset-text--related inset-text--condensed" markdown="1">
Example:

<ul>

>   <li>Captions (called “intralingual subtitles” in some areas) provide content to people who are deaf and hard-of-hearing.</li>

Reviewer 3: ```suggestion
  <li>Captions (called “subtitles” in some areas) provide content to people who are deaf and hard-of-hearing.</li>
```

If a proper noun is singular, it’s not capitalized in the plural.

<aside class="example">
<i>Example:</i><br>

“The Government is focussed on social policy, but previous governments focussed on economic policy” — here, “the Government” is a proper noun because it refers to the current national government.
</aside>

#### Nouns that are capitalized at W3C

##### WCAG levels

> Treat WCAG levels as proper nouns.

Reviewer 3: Don't know that I would specifically call that out so much as just point out the way to do it

> Treat WCAG levels as proper nouns.

Reviewer 3: Removing the section on WCAG Levels.

1ed36b1d111ef1ad15cb846e2f8ce1909c418dce
